proto/ojo/oracle/v1/genesis.proto

```diff
@@ -25,7 +25,7 @@ message GenesisState {
       [ (gogoproto.nullable) = false ];
   repeated PriceStamp medians = 7 [ (gogoproto.nullable) = false ];
   repeated PriceStamp historic_prices = 8 [ (gogoproto.nullable) = false ];
-  repeated PriceStamp medianDeviations = 9 [ (gogoproto.nullable) = false ];
+  repeated PriceStamp median_deviations = 9 [ (gogoproto.nullable) = false ];
```

Comment:
nice. does this change the API for @dixitaniket 's relayer?

Reply:
I don't think it should but I'd like him to confirm just to make sure